server/static/styles.css

```diff
@@ -15,18 +15,91 @@ body {
     justify-content: flex-start;
     align-items: center;
     width: 80%;
-    background-color: #ffffff;
+    background-color: "white";
     padding: 10px;
     border-radius: 5px;
     box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.2);
     margin-top: 10px;
     margin-bottom: 30px;
 }
 
+#top-container{
+    display: grid;
+    grid-template-columns: auto auto auto;
+    column-gap: 3px;
+    width: 100%;
+}
+
 #login-info {
+    display: inline-block;
+    text-align: center;
+}
+
+#switch-box {
```

Comment:
ChatGPT? :)

Would be good if you can change "switch-box" to reference dark-mode explicitly ("#darkmode-switch-box").
As an aside, it's a bit ugly - may have been nicer to have it smaller and entirely above the "logged in as" part (still right aligned), as it hurts the center alignment of the login part and statistics otherwise.

Not a blocker but if you feel like fixing this in a follow-up commit, I will be grateful. :)